### PR TITLE
Cut dependencies from installation page

### PR DESCRIPTION
**Pull request by palewire**
This is of course subjective but my take is that introducing the dependencies list at this point is too much to throw at a total n00b and not really what they're on the page to get. I propose cutting it.

### CODE REVIEW

File: doc/getting_started/installation.rst [hunk 1]
```diff
@@ -67,18 +49,7 @@ development version directly from GitHub using:
```
Comment (joelostblom) [inline doc/getting_started/installation.rst:49]:
```suggestion
Please see `CONTRIBUTING.md <https://github.com/altair-viz/altair/blob/master/CONTRIBUTING.md>`_
for details on how to contribute to the Altair project.

```

File: doc/getting_started/installation.rst [hunk 2]
```diff
@@ -21,26 +22,7 @@ and execute any of the code from the :ref:`example-gallery`.
...
-- importlib_metadata_ (python<3.8)
-- typing_extensions_ (python<3.11)
-
-To run Altair's full test suite and build Altair's documentation requires a few
-additional dependencies, see `CONTRIBUTING.md <https://github.com/altair-viz/altair/blob/master/CONTRIBUTING.md>`_
-for the details.
-
-Development Install
+Install source code
 ===================
```
Comment (joelostblom) [inline doc/getting_started/installation.rst:26]:
```suggestion
Development Installation
========================
```

File: doc/getting_started/installation.rst [hunk 3]
```diff
@@ -2,8 +2,9 @@
 
 .. _installation:
 
-Installation
-============
+Install package
+===============
```
Comment (joelostblom) [inline doc/getting_started/installation.rst:6]:
```suggestion
Installation
============
```

**Review by joelostblom — CHANGES_REQUESTED** [review]
I agree that we can remove the dependencies section from this page, but I think we should keep the headings as is since installing the source code also installs the package and it is more conventionally referred to as a development install

**Comment by palewire** [comment]
Thanks. I've accepted your changes. Or at least tried to.

**Review by joelostblom — APPROVED** [review]
Looks great, thank you!

**Pull request merged**